WordPress/src/main/res/xml/stats_widget_info.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<appwidget-provider
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:initialLayout="@layout/stats_widget_layout"
+    android:minHeight="40dp"
+    android:minWidth="310dp"
+    android:previewImage="@drawable/stats_widget"
```

Comment:
Can we rename `stats_widget.png` to `stats_widget_preview.png` so it's more obvious what it's for?